src/chainstate/coordinator/mod.rs

```diff
@@ -120,6 +120,9 @@ pub trait BlockEventDispatcher {
         winner_txid: Txid,
         matured_rewards: Vec<MinerReward>,
         matured_rewards_info: Option<MinerRewardInfo>,
+        prev_burn_block_hash: BurnchainHeaderHash,
+        prev_burn_block_height: u32,
+        prev_burn_block_timestamp: u64,
```

Comment:
Rather than `prev_*`, I'd call these `parent_*` -- this isn't necessarily the immediately prior burn block: it's the burn block that selected this block's parent.